src/pages/game/Game.tsx

```diff
@@ -1,14 +1,39 @@
-import React, { FC, ReactElement } from 'react';
-import { Link } from 'react-router-dom';
+import React, { FC, ReactElement, useState } from 'react';
 
 import GameCanvas from 'components/gameCanvas/GameCanvas';
-import { ROUTE_CONSTANTS } from 'constants/routeConstants';
+import { Menu, TMenuItem } from 'components';
+import { MENU_ITEMS, MENU_ITEMS_PAUSE, MENU_ACTIONS } from 'constants/menuConstants';
+
+import './game.scss';
 
 const Game: FC = (): ReactElement => {
+  const [ isMenuShown, setIsMenuShown ] = useState<boolean>(true);
+  const [ menuItems, setMenuItems ] = useState<TMenuItem[]>(MENU_ITEMS);
+  const [ menuAction, setMenuAction ] = useState<string | null>(null);
+
+  const toggleMenu = () => {
+    setIsMenuShown(prevValue => !prevValue);
+  };
+
+  const handleMenuAction = (action: string) => {
+    if (action) {
+      setMenuAction(action);
+      if (action === MENU_ACTIONS.GAME_START) {
```

Comment:
❗️
Перед условиями лучше делать отбивку пустой строкой
```suggestion
      setMenuAction(action);
      
      if (action === MENU_ACTIONS.GAME_START) {
```